src/services/publish.service.ts

```diff
@@ -38,6 +39,100 @@ export class PublishService extends Repository {
       configureOptions.posting_latitude = lat.toString();
       configureOptions.posting_longitude = lng.toString();
     }
-    return await this.client.media.configure(configureOptions);
+    return await this.client.media.configureTimeline(configureOptions);
+  }
+
+  public async story(options: PostingStoryOptions) {
+    const uploadedPhoto = await this.client.upload.photo({
+      file: options.file,
+    });
+    const imageSize = await sizeOf(options.file);
+    const configureOptions: MediaConfigureStoryOptions = {
+      upload_id: uploadedPhoto.upload_id,
+      width: imageSize.width,
+      height: imageSize.height,
+      configure_mode: 1,
+    };
+
+    // check for directThread => no stickers supported
+    if (options.threadIds !== undefined) {
```

Comment:
Will be very convenient to have `DirectThreadEntity.broadcastStory  or smth` method.